<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CfgSettingAutomaticMixerGainSharing</class>
 <widget class="QWidget" name="CfgSettingAutomaticMixerGainSharing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>155</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="title">
        <string>Setup</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <item row="4" column="1">
         <widget class="ConfigGainWidget" name="Slider_AttackTime" native="true">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="MaxValue" stdset="0">
           <double>100.000000000000000</double>
          </property>
          <property name="MinValue" stdset="0">
           <double>5.000000000000000</double>
          </property>
          <property name="ParamType" stdset="0">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Detector Time (ms)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Release Time (ms)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="ConfigGainWidget" name="Slider_Noise" native="true">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="MaxValue" stdset="0">
           <double>0.000000000000000</double>
          </property>
          <property name="MinValue" stdset="0">
           <double>-40.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Depth (dB)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>ThresholdLevel (dB)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="ConfigGainWidget" name="Slider_DetectorTime" native="true">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="MaxValue" stdset="0">
           <double>100.000000000000000</double>
          </property>
          <property name="MinValue" stdset="0">
           <double>1.000000000000000</double>
          </property>
          <property name="ParamType" stdset="0">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="ConfigGainWidget" name="Slider_Depth" native="true">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="MaxValue" stdset="0">
           <double>0.000000000000000</double>
          </property>
          <property name="MinValue" stdset="0">
           <double>-40.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Attack Time (ms)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Hold Time (ms)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Noise (dB)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="ConfigGainWidget" name="Slider_ThreadLevel" native="true">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="MaxValue" stdset="0">
           <double>0.000000000000000</double>
          </property>
          <property name="MinValue" stdset="0">
           <double>-80.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="ConfigGainWidget" name="Slider_HoldTime" native="true">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="MaxValue" stdset="0">
           <double>10000.000000000000000</double>
          </property>
          <property name="MinValue" stdset="0">
           <double>10.000000000000000</double>
          </property>
          <property name="StepMode" stdset="0">
           <number>1</number>
          </property>
          <property name="ParamType" stdset="0">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="ConfigGainWidget" name="Slider_ReleaseTime" native="true">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="MaxValue" stdset="0">
           <double>10000.000000000000000</double>
          </property>
          <property name="MinValue" stdset="0">
           <double>5.000000000000000</double>
          </property>
          <property name="StepMode" stdset="0">
           <number>1</number>
          </property>
          <property name="ParamType" stdset="0">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Mix</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="spacing">
         <number>5</number>
        </property>
        <item row="0" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Out Gain (dB)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1" alignment="Qt::AlignHCenter">
         <widget class="ConfigGainWidget" name="Slider_MixOutGain" native="true">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="MaxValue" stdset="0">
           <double>10.000000000000000</double>
          </property>
          <property name="MinValue" stdset="0">
           <double>-100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Out Mute</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="PB_MixOutMute">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Input</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>7</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QScrollArea" name="scrollArea">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1852</width>
           <height>492</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_Hold_2">
            <property name="text">
             <string>Applied
Gain</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_1" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_2" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_3" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_4" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_5" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_6" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_7" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_8" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_9" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_10" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_11" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_12" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_13" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_14" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_15" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_16" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_17" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_18" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_19" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_20" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_21" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_22" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_23" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_24" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_25" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_26" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_27" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_28" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_29" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_30" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_31" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ConfigGainSharingChannel" name="widget_32" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>400</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConfigGainWidget</class>
   <extends>QWidget</extends>
   <header location="global">customWidget/configgainwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigGainSharingChannel</class>
   <extends>QWidget</extends>
   <header>customWidget/configgainsharingchannel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
